src/transaction-flow/input/RevokePermissions/RevokePermissions-flow.tsx

```diff
@@ -88,6 +90,7 @@ export type View =
   | 'revokePermissions'
   | 'revokeChangeFuses'
   | 'revokeChangeFusesWarning'
+  | 'lastWarning'
```

Comment:
I think this is where there was prior confusion about the onSubmit vs onIncrement events in the ControlledNextButton component. The reason for the difference was that if CANNOT_BURN_FUSES was burned, then it would increment to the revokeChangeFusesWarning but if it wasn't then it would skip and go to submit the transaction.